samcli/commands/build/build_context.py

```diff
@@ -70,6 +70,7 @@ def __init__(
         aws_region: Optional[str] = None,
         create_auto_dependency_layer: bool = False,
         stack_name: Optional[str] = None,
+        print_message: bool = True,
```

Comment:
nit: `print_message` seems too generic, maybe we can use `print_success_message` as the variable name?